### ISSUE TITLE
Switch to last active group

### ISSUE THREAD

**Issue by CIAvash** [Quicksaver/Tab-Groups#328]
Add a keyboard shortcut for switching to last active tab group, like the "Switch to last active tab" shortcut in Tab Mix Plus but for tab groups.

**Comment by leggewie** [comment]
I think #181 is a better and more complete request of basically the same thing.

**Comment by Quicksaver** [comment]
Not really, the two would serve very different use-cases, and could probably even complement each other. They're not mutually exclusive at least.

**Comment by jmain33** [comment]
Also related to https://github.com/Quicksaver/Tab-Groups/issues/238, just a different implementation.
